sound/soc/sof/ipc4-topology.c

```diff
@@ -1766,9 +1769,6 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 		       gtw_cfg_config_length,
 		       &ipc4_copier->dma_config_tlv, dma_config_tlv_size);
 
-	/* update pipeline memory usage */
-	sof_ipc4_update_resource_usage(sdev, swidget, &copier_data->base_config);
-
```

Comment:
This fixes fresh unreported issue where driver could send 0 CPC even if proper data found in manifest for copier.

Reply:
@plbossart, the firmware will receive the previously found CPC value for the copier instance:
1. We find X as a matching CPC but CPC of 0 is sent
2. We find Y as a matching CPC but CPC of X is sent
3. We did not find matching CPC (so it should be 0) but CPC of Y is sent

This is only affecting the copier because the `sof_ipc4_update_pipeline_mem_usage()` was after the memcpy and I only did a rename in patch 3f1e22fd066885a0da7a42b6841d5502e0e4ce1b
In other places it was either before the memcpy or there were no memcpy at all.